tests/helics/system_tests/TimingTests2.cpp

```diff
@@ -685,102 +685,6 @@ TEST_F(timing_tests2, offset_timing)
     vFed2->disconnect();
 }
 
-TEST_F(timing_tests2, wait_for_current_time_iterative)
-{
-    extraBrokerArgs = "--debugging";
-    auto broker = AddBroker("test", 2);
-    extraCoreArgs = "--debugging";
-    AddFederates<helics::ValueFederate>("test", 1, broker, 1.0);
-    AddFederates<helics::ValueFederate>("test", 1, broker, 1.0);
-
```

Comment:
Is removing this test intentional, or trying to reduce the noise from this test failing?

Reply:
Ah, it got moved to `iterationTests.cpp`.